### PR TITLE
add deeplow and simon for Qubes Forum work

### PR DESCRIPTION
**Pull request by mfc**

### CODE REVIEW

**Review by marmarek — APPROVED** [review]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Approved as of bb5a95dd1cd4a8319ad0e5253851d5343656fbd2
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCAAdFiEEnYWYU8Ll1fIX9iss8yiUvpaEk4oFAmCQEwEACgkQ8yiUvpaE
k4oi4Q/+I9sWQWxxuMolHkEzNzNI9iXQWYg6nAD09wZNBy/VWrXePeQ5NYD1gjOA
mcXYBqSSKU+zbpwTHokoPX++BW7tq+HqN4R3V1FXk3oiarl95OEalQzdPMJuBuvB
nfryy36vdH9NF3uSYDxhbq3+5i+iVTSL7i1LZxOgqVtAiUuNwmhuyqnXxEvHcQmu
1EfaHJ7XXfhZ/Zhb22oRn66+KeNCYBPtWkqJLmuO46UMyUesLPFkm5uXZ+nZso/K
C65e+Ttc7HKeA+cxZHIYWShQF80Yf2QHtDx9btwqm30RHKBz6ev8T9816KdwTYhW
b0DrP1XOuLQrNI8fOnvXVZMhEQ0ejZP4UjiabQq0S4ZN2ixBZ0+c8/464BfKd8jq
IwvqFFRKt8AeTc8+h4DQWrGYh17wS94vZk4Bx8ZUVMInxxu2GmiWUuV2ZyDSRBIm
fOz1XqZ6CBVSCc7Z41Vwd9Py7gWkqHJGmNy3lVvAdqaJxwf5MxWVoPCQHeS+6oMs
w0aP+jmL4wMAVcct5snw+vb/4p4symziM5QKM7FXtIkNGe8Mi7VqzdwrxDIZr32x
BVzkNXni6nBzuCJsNI2X0SoEuJIj58O/LT3QkqUPEN6wT65L4PDzdjrchicWt9Dd
gHwknHEEO1VpPhKK2NIqE9jdY6CRoi8hH3aXvL/lMIE9cgdqIPs=
=WiDp
-----END PGP SIGNATURE-----
```

File: _data/team.yml [hunk 1]
```diff
@@ -168,6 +168,12 @@
   type: community
   fingerprint: 
 
+- name: Deeplow
```
Comment (deeplow) [inline _data/team.yml:171]: @mfc thanks! Can the name start with lower case ("deeplow"), since it's a username? (like unman or awokd)

Comment (mfc) [reply _data/team.yml]: sure , done!

File: _data/team.yml [hunk 2]
```diff
@@ -234,6 +240,12 @@
   type: community
   fingerprint: 
 
+- name: Simon Newton
+  email: simon.newton@gmail.com
```
Comment (shamen123) [inline _data/team.yml:244]: @mfc  Can we use my email theplexus@ctemplar.com please? sorry to be a pain ;)

**Review by shamen123 — CHANGES_REQUESTED** [review]

File: _data/team.yml (hunk 2 above)
Comment (mfc) [reply _data/team.yml]: yep done, apologies for using wrong address!

**Pull request merged**